ios/RNSConvert.h

```diff
@@ -19,6 +19,12 @@
 + (RNSScreenSwipeDirection)RNSScreenSwipeDirectionFromCppEquivalent:
     (facebook::react::RNSScreenSwipeDirection)swipeDirection;
 
++ (RNSScreenDetentType)RNSScreenDetentTypeFromAllowedDetents:
+    (facebook::react::RNSScreenSheetAllowedDetents)allowedDetents;
+
++ (RNSScreenDetentType)RNSScreenDetentTypeFromLargestUndimmedDetent:
+    (facebook::react::RNSScreenSheetLargestUndimmedDetent)detent;
+
```

Comment:
Codegen generates two different types of enums despite the fact, that only single type is used in JS component spec, thus 2 distinct conversion methods are required.